packages/create-guten-block/app/run.js

```diff
@@ -42,6 +43,7 @@ module.exports = async() => {
 		) }`
 	);
 	await createPluginDir( blockName );
+	await createGitignore( blockDir );
```

Comment:
I think `await createGitignore( blockDir );` should be the part of `createPluginDir` since `run.js` is a high level over view. Can you modify it to call it right after templates are copied in that file instead of this?